nilearn/externals/tempita/__init__.py

```
@@ -38,9 +38,9 @@ def foo(bar):
     from io import StringIO
     from html import escape as html_escape
 except ImportError:
-    from urllib import quote as url_quote
-    from cStringIO import StringIO
-    from cgi import escape as html_escape
+    from urllib import quote as url_quote # type: ignore
```

TODO

find a way to silence all errors in `externals`